Dockerfile

```diff
@@ -19,7 +19,11 @@ RUN export GOOS=$TARGETOS && \
 
 FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.6.0 AS nmi
 # upgrading liblz4-1 due to CVE-2021-3520
-RUN clean-install ca-certificates liblz4-1
+# upgrading libgcrypt20 due to CVE-2021-33560
+# upgrading libgnutls30 due to CVE-2021-20231, CVE-2021-20232, CVE-2020-24659
+# upgrading libhogweed4 due to CVE-2021-20305, CVE-2021-3580
+# upgrading libnettle6 due to CVE-2021-20305, CVE-2021-3580
+RUN clean-install ca-certificates liblz4-1 libgcrypt20 libgnutls30 libhogweed4 libnettle6
```

Comment:
does buster-1.6.3 cover some of these?

Reply:
It only fixes CVE-2021-3520. Let me make the change.

Reply:
Also noticed buster-v1.6.4, which was released yesterday, did not cover these vulnerabilities